spec/unit/node/immutable_collections_spec.rb

```diff
@@ -73,7 +73,11 @@
   end
 
   it "should create a YAML string with content" do
-    expect(copy).to eq(parsed_yaml)
+    # Roundtrip the test string through YAML to compensate for some changes in libyaml-0.2.5
+    # See: https://github.com/yaml/libyaml/pull/186
+    expected = YAML.dump(YAML.load(parsed_yaml))
```

Comment:
This is not great, but the alternative was to modify the value being tested like `copy.split("\n").map(&:rstrip).join("\n")` that seemed more likely to have a strange side effect.